src/components/first-section/first-section.tsx

```diff
@@ -0,0 +1,38 @@
+import { HandCoins } from '@phosphor-icons/react'
+import {
```

Comment:
Só sugestão, para não sujar demais os imports poderia subir como
```suggestion
import * as S from './first-section.styles'
```

Ajuda caso os ux comecem a seccionar mais internamente